superset-frontend/src/components/ErrorMessage/ErrorAlert.tsx

```diff
@@ -123,7 +123,7 @@ export default function ErrorAlert({
         {!isExpandable && !description && (
           <span
             role="button"
-            tabIndex={0}
+            tabIndex={-1}
```

Comment:
Why are we setting tab index to -1 (here and other places)? Doesn't it have a negative effect on accessibility, since you can't navigate to it with tab anymore?

Reply:
Even when navigating to the element, there were no onKeyDown handlers, so you wouldn't be able to do anything after you tabbed to the element, so I removed the ability to tab, because it was out of scope, but I just added the error modal tab functionality since I was here. :)

https://github.com/apache/superset/assets/5186919/6141f4da-c474-47ba-a64e-9bc652b072ec